src/components/ViewSource/ViewSource.js

```diff
@@ -51,6 +64,30 @@ const ViewSource = ({
   const [marc, setMarc] = useState();
   const [isMarcLoading, setIsMarcLoading] = useState(true);
 
+  const redirectToMARCEdit = useCallback(() => {
+    const urlId = isHoldingsRecord ? `${instanceId}/${holdingsRecordId}` : instanceId;
+    const pathname = `/inventory/quick-marc/edit-${isHoldingsRecord ? 'holdings' : 'bib'}/${urlId}`;
+
+    const searchParams = new URLSearchParams(location.search);
+
+    searchParams.delete('relatedRecordVersion');
+    searchParams.append('shared', instance.shared?.toString());
```

Comment:
This is the second place we push `isShared` to URL. It might be worth to make a separate function for this. 
I think that if we open a locally shared record using the shortcut, we will push `isShared` value to false. If we are going to fix that behavior like 
```
const isInstanceShared = Boolean(isShared || isInstanceShadowCopy(instance?.source));
```
 then it would be nice to do it in just one place.